minutes/main/20251015/Readme.md

```diff
@@ -0,0 +1,148 @@
+… I have created another issue a couple of weeks ago
+
+sk: what is your feeling towards the binding registry? We have the 6 months extension, is there any chance we could finish this?
+
+ege: I doubt we can go through the full process, like going to the registry stage because we have the pilot process and want to test it with some of the existing bindings we have
+
+sk: do we want to see more protocols like Matter, there were already some ideas. From my point of view looks good. Any questions?
+
+Ben: Publishing the registry: I don't have a problem with it, but technically it is not within the current charter
+
+sk: when we talked about this charter, 2/3 years ago, this registry concept was not existing and there was no policy on it. This came up during the charter. It is a good point and we need to clarify it if it is not showing in the charter. But I would expect that Kaz would have mentioned if there would have been an issue
+
+<McCool> ntd
+
+ege: Ege discusses his comments on Changes to draft charter #1240: remove some specific wording, removed dicvoery and architecture from normative, manage WoT bindings
+
+sk: I think the digital twin topic came from Michael Legally back in the days, his not active anymore
+
+sk: the plan for the next charter and the key focus topics, how to be more efficient in the WoT group
+… it is mainly caused by the many threats we have, in the past we had more than 10 calls in a week. In the past it was justified, creating the WoT, the idea behind it and what needs to be addressed. The picture was not clear in the beginning. The core here is now clear, the TD is the important technology used here. My proposal for the next charter
+
+would be to focus on the WoT TD. Of course the WoT TD cannot be a technology and a task that can exist alone, you need inputs like the bindings and the registry, but topics like discovery, profiles and architecture I am not sure whether we have the men power for it.
+… I am open to discuss this in the new charter, but if I feel we don't have the men power we focus on the TD 2.0 and then to make another charter related to the other aspects. I have the feeling that other working groups do a similar approach focusing on smaller topics.
+… I would like to hear your views on this aspect
+
+Ben: I completly agree that the TD is the core and the number of meetings and the calls we had in the past were not sustainable. The question is which one survives? The problem I have to only focusing on TD and registry is that we never use the vocabulary from HTTP bindings, to adopt bindings we need a better way to figure out how they will work
+
+with the Profiles. I think if we have to wait for 2 years to then start working on Profiles, this will be too long. I would like to see WoT profiles in the next charter. I do see we need to drop Architecture, dicovery, and security merged with other specs.
```

Comment:
```suggestion
Ben: I completly agree that the TD is the core and the number of meetings and the calls we had in the past were not sustainable. The question is, which one survives? The problem I have with only focusing on TD and registry is that we never use the vocabulary from HTTP bindings; to adopt bindings, we need a better way to figure out how they will work with the Profiles. I think if we have to wait for 2 years to start working on Profiles, this will be too long. I would like to see WoT profiles in the next charter. I do see we need to drop Architecture, dicovery, and security merged with other specs.
```